src/libraries/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj

```diff
@@ -19,10 +24,13 @@
     <Compile Include="SatelliteContractVersionAttributeTests.cs" />
     <Compile Include="MissingSatelliteAssemblyException.cs" />
     <Compile Include="ResourceSetTests.cs" />
-    <Compile Include="$(CommonTestPath)System\Drawing\Helpers.cs"
-             Link="Common\System\Drawing\Helpers.cs" />
+    <Compile Include="$(CommonTestPath)System\Drawing\Helpers.cs" Link="Common\System\Drawing\Helpers.cs" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Resources\AToZResx.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>AToZResx.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
```

Comment:
This is what VS generated when I made a new resx file. Any reason to change it to match what the other files were using?

Reply:
It'd be nice to be consistent.